demo_scripts/containerAppsPython/README.md

```diff
@@ -0,0 +1,140 @@
+# Create a Container App On Azure
+Azure Container Apps enables you to run microservices and containerized applications on a serverless platform. 
+Common use cases include: Deploying API endpoints, Hosting background processing applications and more.
+
+# Azure CLI Login
+First we need to check you are logged in to the Azure in the CLI. The following command will check to see if you are logged in. 
+If not it will open a browser and take you through the login steps.
+
+To Login to Az CLI and select a subscription 
+`az login` followed by `az account list --output table` and `az account set --subscription "name of subscription to use"`
+
+To Install Az CLI
+If you need to install Azure CLI run the following command: curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+
+# Prerequisites
+
+Before you can begin you need to follow the prerequisite steps found here
+
+1. Visit https://github.com/asw101/python-fastapi-pypy
+2. Click "Use this template"
+3. Name your repo "serverless-python". 
+    * **If your GitHub repository Private rather than Public in step 3, you will need to click on "Package settings" on the right hand side, scroll down and click "Change visibility" button to make your package public.**
+4. Create a new branch called release
+5. Click on the Actions tab
+6. View the output of the action
+7. Return to the main repo (Code tab)
+8. Click on "serverless-python" under "Packages" on the right hand side
+9. Copy the `docker pull` command which will include the image name
+10. Update the `GITHUB_USERNAME` environment variable below with your GitHub username or organization name.
+    * **Press `b` and run the command `GITHUB_USERNAME="username"` to set variable.**
```

Comment:
What is this? It means nothing in a published document context. Why is it here?

When run in SimDem the use of an environment variable will automatically result in the customer being prompted for a value if no value exists yet.